controller/controller.tf

```diff
@@ -67,42 +9,40 @@ resource "aws_eip_association" "eip_assoc" {
 
 resource "aws_network_interface" "eni-controller" {
   subnet_id = "${var.subnet}"
+
```

Comment:
Most of the changes like this are from running 'terraform fmt' on the file

(We're using the latest release, which is 0.11.2 when this PR was created).